package.json

```diff
@@ -3,9 +3,11 @@
   "version": "1.0.0",
   "description": "Scripts to create Jira links for Scaffolding pages with missing links",
   "main": "index.js",
+  "type": "module",
   "author": "ServiceRocket",
   "license": "ISC",
```

Comment:
nit: We should think about the licensing of the code since this is a public repo - There's a [site](https://choosealicense.com/) that helps with making the decision.

Reply:
I think `ISC` should be ok for public projects. Full license text [here](https://choosealicense.com/licenses/isc/).

Reply:
super nit: Perhaps we can add the LICENSE file to the repo :sunglasses: